sysapps/device_capabilities/cpu_info_provider_android.cc

```diff
@@ -20,6 +20,8 @@ const char kProcLoadavg[] = "/proc/loadavg";
 namespace xwalk {
 namespace sysapps {
 
+void CPUInfoProvider::init() {}
```

Comment:
so, would it be simpler just to add functional code to the Win-specific constructor?